zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/ElasticsearchEngineClient.java

```diff
@@ -117,6 +123,70 @@ public Map<String, IndexMapping> getMappings(
     }
   }
 
+  @Override
+  public void putSettings(
+      final IndexDescriptor indexDescriptor, final Map<String, String> toAppendSettings) {
+    final var request = putIndexSettingsRequest(indexDescriptor, toAppendSettings);
+
+    try {
+      client.indices().putSettings(request);
+    } catch (final IOException e) {
+      final var errMsg =
+          String.format("Index [%s] settings PUT failed", indexDescriptor.getIndexName());
+      LOG.error(errMsg, e);
+      throw new ElasticsearchExporterException(errMsg, e);
+    }
+  }
+
+  @Override
+  public void putIndexLifeCyclePolicy(
```

Comment:
My understanding is in operate they added policy name and min age as configuration because there was only one policy for every operate indices and they only had to name that policy. 

However now the initial implementation will still be one policy for all operate indices, but in the future we would want to possibly change to one policy per app e.g. operate, tasklist, and in that case we would need to name multiple policies.

however I actually do agree with you that string replacing the %MIN_DELETION_AGE handle is not a good solution I will refactor this.